_source/logzio_collections/_log-sources/lambda-extensions.md

```diff
@@ -0,0 +1,213 @@
+
+### Extension deployment options
+
+You can deploy the extension via the AWS CLI or via the AWS Management Console.
+
+</div>
+
+<!--tab:end -->
+
+
+<!-- tab2:start -->
+<div id="cli">
+
+#### Deploying Logz.io logs extension via the AWS CLI
+
+<div class="tasklist">
+
+##### Deploy the extension, dependencies and configuration
+
+If you haven't done it already, [install](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) and [configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) the AWS CLI.
+
+Add the layer to your function and configure the environment variables using the following command:
+
+```shell
+aws lambda update-function-configuration \
+    --function-name <<FUNCTION-NAME>> \
+    --layers <<LAYERS>> \
+    --environment "Variables={<<ENV-VARS>>}"
+```
+
+**Note:** this command overwrites the existing function configuration. If you already have your own layers and environment variables for your function, list them as well.
+
+| Placeholder | Description | Required/Default|
+|---|---|---|
+| `<<FUNCTION-NAME>>` |  Name of the Lambda Function you want to monitor. |Required|
+| `<<LAYERS>>` | A space-separated list of function layers to add to the function's execution environment. Specify each layer by its ARN, including the version.  For the ARN, see the **Lambda extension versions** table in the last tab of this topic. You may need to add another layer that has the extensions dependencies.  For the libraries that your extension requires, see the **Lambda extensions dependencies** table in the last tab. If your function doesn't already has those libraries under `/opt/python`, you'll need to add the dependencies as a layer, too. | |
+| `<<ENV-VARS>>`  | Key-value pairs containing environment variables that are accessible from function code during execution. Should appear in the following format: `KeyName1=string,KeyName2=string`.  For a list of all the environment variables for the extension, see the **Lambda environment variables** table in the last tab.| |
+
+##### Run the function
+
+Use the following command. It may take more than one run of the function for the logs to start shipping to your Logz.io account.
+
+```shell
+aws lambda update-function-configuration \
+    --function-name <<FUNCTION-NAME>> \
+    --layers [] \
+    --environment "Variables={}"
+```
+
+
+<!-- info-box-start:info -->
+This command overwrites the existing function configuration. If you already have your own layers and environment variables for your function, include them in the list.
+{:.info-box.note}
+<!-- info-box-end -->
+
+
+ </div>  
+
+#### Deleting the extension
+
+To delete the extension, its dependencies and environment variables, use the following command:
+```shell
+aws lambda update-function-configuration \
+    --function-name some-func \
+    --layers [] \
+    --environment "Variables={}"
+```
```

Comment:
@YajB For some reason it doesn't look good in the preview. looks like this:
![image](https://user-images.githubusercontent.com/37478344/116103643-76aa5600-a6b8-11eb-9bb6-dae27d329b65.png)
Can you please fix this?

Reply:
Added a line before code snippet.